includes/Shortcodes.php

```diff
@@ -42,4 +42,12 @@ function  weatherShortcode(): string {
                     <h3 style='border-top: solid #6ea7ff; width: 100%; text-align: center; margin: 0; display: grid; place-items: center; height: 80px;'>$weatherDescription</h3>
                 </div>";
     }
+
+	function sunShortcode() {
+		$city = $this->api->getCity();
+		$sunrise = date("Y-m-d H:i:s", $this->api->getSunrise());
```

Comment:
Bro, you can put the format into a variable so that it's easier to change and to test

Reply:
🖕

Reply:
nah bro thats really bad because it's a extra line of code